docs/user-guide/kubectl/kubectl_expose.md

```diff
@@ -72,6 +72,9 @@ kubectl expose rc streamer --port=4100 --protocol=udp --name=video-stream
 
 # Create a service for a replicated nginx using replica set, which serves on port 80 and connects to the containers on port 8000.
 kubectl expose rs nginx --port=80 --target-port=8000
+
+# Create a service for an nginx deployment, which serves on port 80 and connects to the containers on port 8000.
+kubectl expose deployment nginx --port=80 --target-port=8000
 ```
```

Comment:
@johndmulhausen @bgrant0607 What's the process to copy these changes to the docs repo?